torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -285,6 +290,7 @@ def _record_post_forward(self) -> None:
         self._post_forward_indices.append(post_forward_index)
 
     def pre_backward(self, default_prefetch: bool, *unused: Any):
+        logger.debug("%s", self._with_fqn("FSDP::pre_backward"))
```

Comment:
Ah sorry, this is my bad. I think we can move this one to below the early return if already in `TrainingState.PRE_BACKWARD`. (so move this down two lines)